### PR TITLE
Added preference consent button

### PR DESCRIPTION
**Pull request by dyampols**
- Now if user is signed in they will be given the option to opt out of their preference information being taken into account to recommend attractions 
- Note: The checkbox will only appear if the user is signed in, if a user is not signed in the recommend endpoint automatically recommends without taking into account preferences 
- Made changes to old unit tests + unit tested changes

### CODE REVIEW

File: travvy/src/services/AttractionsService.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import axios from 'axios'
 
 export default {
   // It makes get request to recommend attractions based on city
-    recommend(city,groupSize,startDate,endDate,user) {
-        return axios.get('/recommend', {params:city,groupSize,startDate,endDate,user})  
+    recommend(city,groupSize,startDate,endDate,user,preference) {
+        return axios.get('/recommend', {params:city,groupSize,startDate,endDate,user,preference})
```
Comment (emilynairnn) [inline travvy/src/services/AttractionsService.js:7]: What is "preference" referring to here?

**Review by emilynairnn — APPROVED** [review]

**Comment by emilynairnn** [comment]
I think this is a great feature for our users! Nice work on the implementation.

File: travvy/src/services/AttractionsService.js (hunk 1 above)
Comment (dyampols) [reply travvy/src/services/AttractionsService.js:7]: It refers to the value that is "Yes" if the user would like their preferences ignored. If its "Yes" the back end logic is triggered to recommend attractions without the preferences being taken into account.

Comment (lucasungaretti) [reply travvy/src/services/AttractionsService.js:7]: that makes sense! thanks for explaining

**Review by lucasungaretti — APPROVED** [review]

**Comment by lucasungaretti** [comment]
looks good! just have to take care of the conflicts!

**Pull request merged**